src/Client/README.md

```diff
@@ -1,10 +1,25 @@
-# ConnectWise PHP Client
+# SPINEN's ConnectWise PHP Client
```

Comment:
It looks like each of these folders (Client, Library, etc) is its own project. Why don't we make subsplit projects from these, instead of having multiple composer files and READMEs for each one in the same project?

Taylor does this with the Illuminate packages.